lib/check-minor-version.js

```diff
@@ -0,0 +1,18 @@
+var semver = require('semver')
+
+module.exports = function (app) {
+  if (!app || !app.path) return
+
+  require('electron-version')(app.path || 'electron', function (err, electronVersion) {
```

Comment:
I was thinking of this as a "performance optimization" to prevent requiring the module until it's actually needed. I supposed I could move `semver` into that conditional as well.

Do you think it's worth it?